app/src/main/java/org/sopt/and/presentation/ui/main/screen/MainScreen.kt

```diff
@@ -127,23 +138,39 @@ fun MainScreen(
         containerColor = Color.Black,
         contentWindowInsets = WindowInsets(0.dp, 0.dp, 0.dp, 0.dp),
         content = { innerPadding ->
-            // selectedTab 값에 따라 표시되는 화면을 변경
-            when (selectedTab) {
-                MainTabList.HOME -> HomeScreen(innerPadding)
-                MainTabList.SEARCH -> SearchScreen(innerPadding)
-                MainTabList.MY -> MyPageScreen(
-                    paddingValues = innerPadding,
-                    userEmail = userEmail
-                )
+            Box(
+                modifier = Modifier
+                    .fillMaxSize()
+                    .padding(innerPadding)
+            ) {
+                when (selectedTab) {
+                    MainTabList.HOME -> HomeScreen()
+                    MainTabList.SEARCH -> SearchScreen()
+                    MainTabList.MY -> {
+                        when (userHobbyState) {
+                            is UserHobbyState.Success -> {
+                                val userHobby = userHobbyState.hobby
+                                MyPageScreen(userHobby = userHobby)
```

Comment:
```suggestion
                                MyPageScreen(userHobby =  userHobbyState.hobby)
```

이렇게 하면 객체 선언 안해도될거 같아요